src/helm/benchmark/run.py

```diff
@@ -236,6 +242,8 @@ def main():
 
     for huggingface_model_name in args.enable_huggingface_models:
         register_huggingface_model_config(huggingface_model_name)
+    for huggingface_model_path in args.enable_local_huggingface_models:
+        register_huggingface_model_config(huggingface_model_path, local=True)
```

Comment:
Can we make this a different method?

Reply:
I think most of the new method would end up being duplicated -- the only difference is if we call `HuggingFaceModelConfig.from_string` or `HuggingFaceModelConfig.from_path`. Is that okay?